cmd/osbuild-worker/main.go

```diff
@@ -269,7 +269,13 @@ func main() {
 			ProxyURL:     proxy,
 		})
 		if err != nil {
-			logrus.Fatalf("Error creating worker client: %v", err)
+			var logLevel logrus.Level
+			if client == nil {
+				logLevel = logrus.FatalLevel
+			} else {
+				logLevel = logrus.ErrorLevel
+			}
```

Comment:
What's your rationale behind using a different log level based on `client` being `nil`?

Reply:
Because Fatalf terminates the application and if client is nil we really can't continue.
(So a more severe thing than some connection problem)